docs/the-most-common-tasks.md

```diff
@@ -0,0 +1,151 @@
+
+## Force closing games
+
+In some cases the game may fail - sometimes because of an excessive amount of substitutes, sometimes because the game was not started and people still wait for a substitute or for some reason the pickup server does not end the pickup (change pickup state from `started` to `ended`). This is when the game should be force-closed. Click `force end` in order to force-close the game.
+
+:::caution
+Force ended games do not count in the player statistics.
+:::
+
+## Setting up player skills
+
+This is probably the most important task of every single pickup admin. Pickup games on tf2pickup.org are based on skills - values assigned to players in order to define their skill on certain classes, depending on a gamemode used on a website. These are integer values between [-9007199254740991 and 9007199254740991](https://www.tektutorialshub.com/typescript/typescript-number-min-max-safe-values/). Any range of values can be used in order to define skills for players. These skill values are used by the pickup site in order to balance out teams for the pickup games, so after all people add and ready up in the queue, the pickup server will calculate the most balanced teams based on skill values of players who added up on certain classes.
+
+### Balancing target
+
+In short, the pickup admin team is to define skill range for classes, set them on every single player and observe game results after setting those skills up. If the games were unbalanced, admins should review the game, see who made an inpact in the game result and adjust skill values, so the next game will be more balanced. The best results are achieved when the games are close or even if they end up in a draw.
+
+[This](https://tf2pickup.fi/game/60cf5a2b6d2b980013b258e0) is an example of a balanced game (it ended up with a score 1:2 for the RED team):
+
+![game-skill-values](/img/content/game-skill-values.png)
+
+If you add up individual player values per team, you will end up with this:
+
+|  | BLU | RED |
+|:-:|:-:|:-:|
+|  | 165 | 40 |
+|  | 210 | 180 |
+|  | 145 | 150 |
+|  | 120 | 115 |
+|  | 110 | 180 |
+|  | 90 | 170 |
+| Result | 840 | 835 |
+
+This is pretty much expected result.
+
+When launching a new site, it's a good practise to make a list of players potentially playing on the site in the future in a spreadsheet with their SteamID64 values next to their nicknames, assign skills to them with your pickup admin team and before the site launch, force create accounts for everybody on the list and assign skills you've decided on. In that way there is a smaller risk that the games may be less balanced. Feel free to use [this](https://docs.google.com/spreadsheets/d/1mCG29TwUoqZhzTQz4UslekaRozSr2Sud5nY26RzJU2U/edit?usp=sharing) spreadsheet as an example of how to define skills for 6v6 players and [this](https://docs.google.com/spreadsheets/d/1zsoYdz32iQwRGVWK5orxSjcqs1e5NJ2ViOLV7JjBu3g/edit?usp=sharing) one for Highlander.
```

Comment:
Technical: please, remove this section. Force-creating player accounts is not designed for this purpose. The skill import/export tool will be available soon.